### ISSUE TITLE
[4.x]: Image transform returns null for GIFs when transformGifs is disabled

### ISSUE THREAD

**Issue by carlcs** [craftcms/cms#13808]
### Description

This is a follow-up to https://github.com/craftcms/cms/issues/13624.

### Steps to reproduce

1. Set `transformGifs` to `false`
2. Run the following GraphQL queries to query for a GIF asset and transform it
    ```
    query a {
      asset(id: "3") @transform(width: 300) {
        url
      }
    }

    query b {
      asset(id: "3") {
        url @transform(width: 300)
      }
    }
    ```

### Expected behavior

Query A and B return the URL of the GIF.

### Actual behavior

Query A returns the URL of the GIF, but Query B returns `null` for the URL.

```
{
  "data": {
    "asset": {
      "url": null
    }
  }
}
```

### Craft CMS version

4.5.6.1

### Installed plugins and versions

n/a

**Comment by i-just** [comment]
Thanks again for raising a new issue for this and all the info! I raised a PR for this.

**Comment by brandonkelly** [comment]
Fixed via #13815!

**Comment by brandonkelly** [comment]
Craft 4.5.7 is out now with the fix for this. Thanks again for reporting!
